app/controllers/items_controller.rb

```diff
@@ -43,14 +45,8 @@ def create
     if result.success?
       redirect_to items_path, notice: "#{result.item.name} added!"
     else
-      @base_items = BaseItem.without_kit.alphabetized
-      # Define a @item to be used in the `new` action to be rendered with
-      # the provided parameters. This is required to render the page again
-      # with the error + the invalid parameters
-      @item = current_organization.items.new(item_params)
-
       flash[:error] = "Something didn't work quite right -- try again?"
-      render action: :new
+      prepare_new_action_render
```

Comment:
Not sure I understand why this was moved? All you need to do here is change `flash[:error]` to be the actual error message instead of a generic "Something didn't work quite right".

Reply:
Makes perfect sense, I initially did that but overthought it and convinced myself it was not that simple... 🥲 thank you! I will get this fixed up right now.